konveyor/pelorus
# Switch documentation to material theme for mkdocs

## Describe the behavior changes introduced in this PR

I propose switching from the default readthedocs theme to the [Material for MkDocs theme](https://squidfunk.github.io/mkdocs-material/). This theme in my opinion makes the documentation look and read much cleaner, as well as includes several built-in UX enhancements we can start to take advantage of, including codeblock captions, annotations, buttons, enhanced icon support, and more.

## Linked Issues?

resolves #<issue number>

related to #<issue number>

## Testing Instructions

Check that the readthedocs PR build passes, and view the demo content via readthedocs or `mkdocs serve`

@redhat-cop/mdt
